packages/fxa-content-server/app/tests/spec/views/connect_another_device.js

```diff
@@ -396,7 +379,6 @@ describe('views/connect_another_device', () => {
 
       it('shows the marketing area, logs appropriately', () => {
         assert.isTrue(view._isSignedIn.called);
-        assert.lengthOf(view.$('#pair-everywhere'), 1);
```

Comment:
I believe this was added recently to get the test to pass.  But if anyone know how can I manually test this in an actual user agent, I'd love to know.

Reply:
Should this check for length of `cad-not-now` or should we check that somewhere else?